cmd/centrifuge/create_config.go

```diff
@@ -41,7 +41,7 @@ func init() {
 				log.Error(err)
 			}
 
-			err = cmd.CreateConfig(targetDataDir, ethNodeURL, accountKeyPath, string(pwd), network, apiHost, apiPort, p2pPort, bootstraps, txPoolAccess, false, "", nil)
+			err = cmd.CreateConfig(targetDataDir, ethNodeURL, accountKeyPath, string(pwd), network, apiHost, apiPort, p2pPort, bootstraps, txPoolAccess, false, "", nil, "")
```

Comment:
Do you want to enable here as well, reading the notification_endpoint extra parameter?

Reply:
I would like to keep the changes to a minimum, We can do it in a future pr that makes changes to commands.